lib/assets/index.js

```diff
@@ -1,5 +1,18 @@
 module.exports = {
-  'icon-32': __dirname + '/icon-32.png',
-  'icon-paused-32': __dirname + '/icon-paused-32.png',
-  'icon-playing-32': __dirname + '/icon-playing-32.png'
+	darwin: {
+		icon: __dirname + '/darwin-icon.png',
+		playing: __dirname + '/darwin-icon-playing.png',
+		paused: __dirname + '/darwin-icon-paused.png'
+	},
+	win32: {
+		// Can use darwin icons, because sizes are the same
+		icon: __dirname + '/darwin-icon.png',
+		playing: __dirname + '/darwin-icon-playing.png',
+		paused: __dirname + '/darwin-icon-paused.png'
+	},
+	any: {
+		icon: __dirname + '/icon.png',
+		playing: __dirname + '/icon-playing.png',
+		paused: __dirname + '/icon-paused.png'
+	}
```

Comment:
Possibly, but it would be a good idea to pre-scale the images for each platform, as many tools introduce artifacts or inconsistencies due to differences in SVG rendering